CMakeLists.txt

```diff
@@ -223,17 +223,17 @@ add_subdirectory(docs)
 
 # DESTINATION gets the previous path and appends the argument --> /opt/local/GrPPI
 
-# cmake -DCMAKE_INSTALL_PREFIX=path/to/folder
+# cmake .. -DCMAKE_INSTALL_PREFIX=path/to/folder
```

Comment:
Such kind of information should be added to the README.md file. Remove from here and explain the the INSTALL.md file